test_conformance/vulkan/test_vulkan_api_consistency_for_1dimages.cpp

```diff
@@ -0,0 +1,206 @@
+#include <vulkan_interop_common.hpp>
+#include <opencl_vulkan_wrapper.hpp>
+#include <vulkan_wrapper.hpp>
+#if !defined(__APPLE__)
+#include <CL/cl.h>
+#include <CL/cl_ext.h>
+#else
+#include <OpenCL/cl.h>
+#include <OpenCL/cl_ext.h>
+#endif
+
+#include <assert.h>
+#include <vector>
+#include <iostream>
+#include <string.h>
+#include "harness/testHarness.h"
+#include "harness/typeWrappers.h"
+#include "harness/deviceInfo.h"
+
+int test_consistency_external_for_1dimage(cl_device_id deviceID,
+                                          cl_context _context,
+                                          cl_command_queue _queue,
+                                          int num_elements)
+{
+    cl_int errNum;
+    VulkanDevice vkDevice;
+
+    // Context and command queue creation
+    cl_platform_id platform = NULL;
+    cl_context context = NULL;
+    cl_command_queue cmd_queue = NULL;
```

Comment:
Can we use `clContextWrapper` and `clCommandQueueWrapper` instead?  Then we won't have to free these objects "manually".